src/components/TodosLogic.jsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-param-reassign */
```

Comment:
It's not advisable to disable the rules of the linter files. See line 54 of this file for a workaround to avoid this.